src/EditorFeatures/Core/FindUsages/AbstractFindUsagesService.cs

```diff
@@ -17,20 +17,22 @@ internal abstract partial class AbstractFindUsagesService : IFindUsagesService
                 document, position, context.CancellationToken).ConfigureAwait(false);
             if (tuple == null)
             {
-                context.ReportMessage(EditorFeaturesResources.Cannot_navigate_to_the_symbol_under_the_caret);
+                await context.ReportMessageAsync(
+                    EditorFeaturesResources.Cannot_navigate_to_the_symbol_under_the_caret).ConfigureAwait(false);
                 return;
             }
 
             var message = tuple.Value.message;
 
             if (message != null)
             {
-                context.ReportMessage(message);
+                await context.ReportMessageAsync(message).ConfigureAwait(false);
                 return;
             }
 
-            context.SetSearchTitle(string.Format(EditorFeaturesResources._0_implementations,
-                FindUsagesHelpers.GetDisplayName(tuple.Value.symbol)));
+            await context.SetSearchTitleAsync(
```

Comment:
If this is the case, I would suggest holding on to the task and deferring the actual await until later. This would cause it to operate as a non-blocking parallel operation without hurting the ability to observe failures.